app/src/main/AndroidManifest.xml

```diff
@@ -1,131 +1,193 @@
 <?xml version="1.0" encoding="utf-8"?>
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-    package="org.gdg.frisbee.android" android:versionCode="27" android:versionName="1.0.10" android:installLocation="auto">
+  package="org.gdg.frisbee.android"
+  android:installLocation="auto">
```

Comment:
Ok, sorry, I misread. Taking it back then. 

On February 8, 2015 9:37:06 PM CET, "Friedger Müffke" <notifications@github.com> wrote:
>> @@ -1,131 +1,193 @@
>>  <?xml version="1.0" encoding="utf-8"?>
>>  <manifest xmlns:android="http://schemas.android.com/apk/res/android"
>> -    package="org.gdg.frisbee.android" android:versionCode="27"
>android:versionName="1.0.10" android:installLocation="auto">
>> +  package="org.gdg.frisbee.android"
>> +  android:installLocation="auto">
>
>the documentation says other: internalOnly is the default
>http://developer.android.com/guide/topics/manifest/manifest-element.html#install
>
>---
>Reply to this email directly or view it on GitHub:
>https://github.com/gdg-x/frisbee/pull/180/files#r24302055

--
Sent from mobile. Please excuse brevity and any typos.